platforms/documentation/docs/src/docs/userguide/dep-man/advanced/variant_aware_resolution.adoc

```diff
@@ -366,6 +367,75 @@ Gradle knows the _consumer_ requires a Java 7-compatible library, but the _produ
 If the _consumer_ requested `org.gradle.jvm.version=15`, Gradle could choose either the Java 8 or Java 11 *variant*.
 Gradle would then select the highest compatible version—Java 11.
 
+[[sec:secondary-variants]]
+== Secondary Variants
+
+**Secondary variants** are *additional sets of artifacts* that a variant can expose, which do **not** participate in graph resolution.
+Instead, they are considered *only during artifact selection*.
+
+A secondary variants:
+
+* Is an additional artifact set exposed by a configuration.
+* It has its own attributes, artifacts, and optionally capabilities.
+* It must be explicitly published (secondary variants are not usually published or declared in metadata).
+* It may be selected via an `ArtifactView` or `ArtifactResolutionQuery`, if the requested attributes match.
+
+Secondary variants are typically used to expose *precomputed transforms* of the main artifact like `.class` files extracted from a JAR, or the `resources` directory.
+
+WARNING: Secondary variants are sometimes referred to in tooling or logs, but this term is being deprecated in favor of more precise concepts like *artifact sets* or *precomputed transforms*.
```

Comment:
For this very reason I don't think we should advertising this feature as "Secondary Variants" 

The title of this section should be "Variant Artifact Sets" Or "Secondary Artifact Sets"

Reply:
Unfortunately, our users and our console output still use "Secondary Variants" so I need that term to be defined and searchable in docs. Maybe if we update our tool I could change it? For now I need to address a number of Github issues about this topic.

I updated the section to be called: == Secondary Variants (Secondary Artifact Sets)